### PR TITLE
MON-20534-refresh-snmp-protocol-topic

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 1]
```diff
@@ -215,206 +211,273 @@ apt install centreon-plugin-applications-protocol-snmp
...
+	--warning= \
+	--critical= \
 	
 ```
 
 La commande devrait retourner un message de sortie similaire à :
 
 ```bash
-OK: System uptime is: 3m 25s | 'uptime'=205s;;;0; 
+OK: |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:272]: empty output

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 2]
```diff
@@ -215,206 +211,273 @@ apt install centreon-plugin-applications-protocol-snmp
...
-| --snmp-tls-trust-cert                      | A trusted certificate to use as trust anchor (like a CA certificate) for verifying a remote server's certificate. If a CA certificate is used to validate a certificate then the TheirHostname parameter must also be specified to ensure their presented hostname in the certificate matches.                                                                                                                                                                                                                                                                             | SNMP   |
-
-
+| Mode                | Modèle de service associé              |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:298]: this table looks suspicious, with "not used" everywhere

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 3]
```diff
@@ -1,65 +1,65 @@
...
-Le connecteur apporte les modèles de service suivants (classés par modèle d'hôte):
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
 
 <Tabs groupId="sync">
-<TabItem value="App-Protocol-SNMP" label="App-Protocol-SNMP">
+<TabItem value="App-Protocol-SNMP-custom" label="App-Protocol-SNMP-custom">
 
 | Alias | Modèle de service | Description |
 |:------|:------------------|:------------|
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:24]: you can't have an empty table here, you need to replace it by a sentence saying that no service templaes are linked to this host template

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 4]
```diff
@@ -1,65 +1,65 @@
...
-| N/A   | N/A               | N/A         |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Protocol-SNMP-custom** est utilisé.
 
 </TabItem>
-<TabItem value="App-Protocol-SNMP-Only" label="App-Protocol-SNMP-Only">
+<TabItem value="App-Protocol-SNMP-Only-custom" label="App-Protocol-SNMP-Only-custom">
 
 | Alias | Modèle de service | Description |
 |:------|:------------------|:------------|
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:32]: same here

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 5]
```diff
@@ -1,53 +1,54 @@
 ---
 id: applications-protocol-snmp
-title: SNMP Protocol
+title: Generic SNMP
 ---
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-## Pack Assets
+## Pack assets
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md:8]: same comments as for the FR version

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 6]
```diff
@@ -1,51 +1,47 @@
...
-| N/A   | N/A               | N/A         |
+ Ce modèle d'hôte n'apporte pas de modèle de service.
 
 </TabItem>
 <TabItem value="App-Protocol-SNMP-Only" label="App-Protocol-SNMP-Only">
 
-| Alias | Modèle de service | Description |
-|:------|:------------------|:------------|
-| N/A   | N/A               | N/A         |
+ Ce modèle d'hôte n'apporte pas de modèle de service.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:25]:
```suggestion
Ce modèle d'hôte n'apporte pas de modèle de service.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 7]
```diff
@@ -1,51 +1,47 @@
...
+* **App-Protocol-SNMP-custom**
+* **App-Protocol-SNMP-Only-custom**
 
 <Tabs groupId="sync">
 <TabItem value="App-Protocol-SNMP" label="App-Protocol-SNMP">
 
-| Alias | Modèle de service | Description |
-|:------|:------------------|:------------|
-| N/A   | N/A               | N/A         |
+ Ce modèle d'hôte n'apporte pas de modèle de service.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:20]:
```suggestion
Ce modèle d'hôte n'apporte pas de modèle de service.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md (hunk 6 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:25]:
```suggestion
Ce modèle d'hôte n'apporte pas de modèle de service.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 8]
```diff
@@ -1,51 +1,47 @@
...
 
-| Alias         | Modèle de service               | Description                                                        |
-|:--------------|:--------------------------------|:-------------------------------------------------------------------|
-| Generic-Value | App-Protocol-SNMP-Numeric-Value | Contrôle permettant de récupérer une valeur numérique d'un OID     |
-| Generic-Value | App-Protocol-SNMP-String-Value  | Contrôle permettant de récupérer une chaine de caractères d'un OID |
-| Uptime        | App-Protocol-SNMP-Uptime        | Contrôle l'uptime d'un équipement en utilisant l'OID standard      |
+| Alias         | Modèle de service                      | Description                                                        |
+|:--------------|:---------------------------------------|:-------------------------------------------------------------------|
+| Generic-Value | App-Protocol-SNMP-Numeric-Value-custom | Contrôle permettant de récupérer une valeur numérique d'un OID     |
+| Generic-Value | App-Protocol-SNMP-String-Value-custom  | Contrôle permettant de récupérer une chaine de caractères d'un OID |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:33]:
```suggestion
| Generic-Value | App-Protocol-SNMP-String-Value-custom  | Contrôle permettant de récupérer une chaîne de caractères d'un OID |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 9]
```diff
@@ -71,11 +68,17 @@ pour en savoir plus sur la découverte automatique de services et sa [planificat
...
-
 ### Métriques & statuts collectés
 
+Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.
+
 <Tabs groupId="sync">
-<TabItem value="Generic-Value" label="Generic-Value">
+<TabItem value="Numeric-Value" label="Numeric-Value">
+
+Coming soon
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:78]: coming soon

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 10]
```diff
@@ -71,11 +68,17 @@ pour en savoir plus sur la découverte automatique de services et sa [planificat
...
+
+</TabItem>
+<TabItem value="String-Value" label="String-Value">
 
 Coming soon
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:83]: coming soon

File: pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 11]
```diff
@@ -70,11 +69,17 @@ and in the [following chapter](/docs/monitoring/discovery/services-discovery/#di
...
 <Tabs groupId="sync">
-<TabItem value="Generic-Value" label="Generic-Value">
+<TabItem value="Numeric-Value" label="Numeric-Value">
+
+Coming soon
+
+</TabItem>
+<TabItem value="String-Value" label="String-Value">
 
 Coming soon
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md:84]: coming soon

File: pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 12]
```diff
@@ -70,11 +69,17 @@ and in the [following chapter](/docs/monitoring/discovery/services-discovery/#di
...
-
 ### Collected metrics & status
 
+Here is the list of services for this connector, detailing all metrics linked to each service.
+
 <Tabs groupId="sync">
-<TabItem value="Generic-Value" label="Generic-Value">
+<TabItem value="Numeric-Value" label="Numeric-Value">
+
+Coming soon
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md:79]: coming soon

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by omercier — APPROVED** [review]

File: pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 13]
```diff
@@ -1,51 +1,48 @@
...
+This host template does not bring any service template.
 
 </TabItem>
-<TabItem value="App-Protocol-SNMP-Only" label="App-Protocol-SNMP-Only">
+<TabItem value="App-Protocol-SNMP-Only-custom" label="App-Protocol-SNMP-Only-custom">
 
-| Service Alias | Service Template  | Service Description |
-|:--------------|:------------------|:--------------------|
-| N/A           | N/A               | N/A                 |
+This host template does not come linked to any service template but it's host check command is based on a SNMP Uptime check, thus valid even when ICMP protocol is blocked.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md:26]:
```suggestion
This host template does not bring any service template but its host check command is based on an SNMP uptime check, thus valid even when the ICMP protocol is blocked.
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 14]
```diff
@@ -217,51 +220,64 @@ apt install centreon-plugin-applications-protocol-snmp
...
 
+3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
+
 ## How to check in the CLI that the configuration is OK and what are the main options for?
 
 Once the plugin is installed, log into your Centreon poller's CLI using the
-**centreon-engine** user account (`su - centreon-engine`) and test the plugin by
-running the following command:
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector
+is able to monitor a server using a command like this one (replace the sample values by yours):
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-snmp.md:271]:
```suggestion
is able to monitor a resource using a command like this one (replace the sample values by yours):
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md [hunk 15]
```diff
@@ -215,206 +216,261 @@ apt install centreon-plugin-applications-protocol-snmp
...
 
+3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparaît dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.
+
 ## Comment puis-je tester le plugin et que signifient les options des commandes ?
 
 Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
 de commande depuis votre collecteur Centreon en vous connectant avec
-l'utilisateur **centreon-engine** (`su - centreon-engine`) :
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser un serveur en utilisant une commande
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-snmp.md:268]:
```suggestion
que le connecteur arrive bien à superviser une ressource en utilisant une commande
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**